awx/ui/package.json

```diff
@@ -40,6 +40,7 @@
     "@lingui/loader": "^3.8.3",
     "@lingui/macro": "^3.7.1",
     "@nteract/mockument": "^1.0.4",
+    "@wojtekmaj/enzyme-adapter-react-17": "0.6.5",
```

Comment:
the fact that enzyme seems to be falling behind is a little concerning. I wonder if we're going to have to evaluate the switch to react testing library at some point 😖

Reply:
@keithjgrant enzyme adapter still does not have support for react 17. It seems like react testing is being kept in sync a bit about react versions. Something to be evaluated IMO.
https://testing-library.com/docs/react-testing-library/migrate-from-enzyme/